atef/widgets/config/data_active.py

```diff
@@ -982,8 +983,11 @@ class ActionRowWidget(TargetRowWidget):
     setting_button: QtWidgets.QToolButton
     curr_val_thread: Optional[BusyCursorThread]
 
+    edit_widget = ClassVar[Optional[QtWidgets.QWidget]]
```

Comment:
```suggestion
    edit_widget: ClassVar[Optional[QtWidgets.QWidget]] = None
```

Typo - I've done this one frequently too

Should it be a classvar though? You access it as `self.edit_widget` below

Reply:
ah good point.  should just be an attribute